02.general/19.dns-filtering-syntax/docs.en.md

```diff
@@ -0,0 +1,21 @@
+---
+title: DNS fitlering rules syntax
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+
+Some AdGuard products provide DNS ad blocking feature. To learn more about the principle of DNS filtering, read [this KB article](https://kb.adguard.com/en/general/dns-filtering-android). Compared to the [traditional ad blocking](https://kb.adguard.com/en/general/how-ad-blocking-works), DNS filtering is more "crude" and allows less customization. It doesn't support the complex [syntax](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters) we use in our filters, but it does support simplified syntax that allows you to block certain domains.
```

Comment:
Maybe we should list products?

Reply:
It's only about Adguard/Home, does it makes sense?

Reply:
it will grow so of course it does